Source/JavaScriptCore/runtime/DateConstructor.h

```diff
@@ -52,4 +52,37 @@ STATIC_ASSERT_ISO_SUBSPACE_SHARABLE(DateConstructor, InternalFunction);
 JSObject* constructDate(JSGlobalObject*, JSValue newTarget, const ArgList&);
 JSValue dateNowImpl();
 
+// https://tc39.es/ecma262/#sec-makeday
+inline double makeDay(double year, double month, double date)
```

Comment:
These probably ought to be `static inline` to avoid double definitions in case the compiler chooses not to inline them.
```suggestion
static inline double makeDay(double year, double month, double date)
```